### DISCUSSION TITLE
New to coding and would love to become a pro

### DISCUSSION THREAD

**Discussion by ghost** [codewars/codewars.com#3037] [Q&A]
Hi, i am new to coding and having difficulty answering the basic questions here on Cod Wars. Does anyone have a plan of action or even recommended courses they could provide? this information would be beneficial thank you in advance.

**Comment by hobovsky** [comment]
Please note that Github Discussions are meant for discussing matters related to the  features and functionalities of the Codewars platform. Codewars Github is not meant to discuss general programming topic, or for general coding help, learning, etc.

For advice on learning, you can try Codewars Discord and one of its related channels (`beginners`, `help-solve`, or a language-specific channel like `JavaScript`, or some other which you feel fits better).

I am closing this discussion, and going to delete it soon as off-topic.
